ig/input/fsh/nbs-obr-segment.fsh

```diff
@@ -14,8 +14,7 @@ Description: "The OBR Segment"
 * universalServiceIdentifierNameOfCodingSystem 1..1 SU string "OBR-4.3 Universal Service Identifier Name Of Coding System"
 * universalServiceIdentifierOriginalText 1..1 SU string "OBR-4.9 Universal Service Identifier Original Text"
 * observationDateTime 1..1 SU dateTime "OBR-7 Observation Date/Time"
-// other optional fields
-
+* specimenActionCode 0..1 SU string "OBR-11 Specimen Action Code (value is always 'P')"
```

Comment:
Since this is _always_ `P`, should we change the cardinality to be `1..1`?

```suggestion
* specimenActionCode 1..1 SU string "OBR-11 Specimen Action Code (value is always 'P')"
```

Reply:
Good catch